src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -248,6 +275,12 @@ MessagePtr TwitchMessageBuilder::build()
     // words
     QStringList splits = this->originalMessage_.split(' ');
 
+    if (this->thread_ && !splits.isEmpty() && splits[0].startsWith('@'))
+    {
+        // If this is a reply, remove the first word which should be "@username"
+        splits.pop_front();
+    }
+
```

Comment:
I'm 100% against modifying actual message content. I want to be able to get the exact content of the message, not get lied to. this only complicates things with bots, mentions, selecting whole message text... besides, if users would try to execute bot commands or send a banphrase message, they'd get confused why bot doesn't react because they'd think that a message starts with "foo", not "@username foo", but in reality the latter is the actual beginning of the message content which I believe would cause issues if changed

Reply:
I agree with your reasoning. I also found that removing the username messes up Twitch emote rendering, so this will be removed.

Reply:
306f7a7